app/src/main/java/com/chatgptlite/wanted/constants/Constants.kt

```diff
@@ -9,3 +9,4 @@ const val urlToGithub = "https://github.com/lambiengcode"
 
 const val matchResultString = "\"text\":"
 const val matchResultTurboString = "\"content\":"
+const val ConversationTestTag = "ConversationTestTag"
```

Comment:
follow camelCase

Reply:
Haha .... Schoolboy error!! I have not even noticed it.